source/expectations/ut_expectation.tpb

```diff
@@ -719,5 +694,37 @@ create or replace type body ut_expectation as
     self.not_to( ut_contain(a_expected).negated() );
   end;
   
+  member function  to_be_within(a_dist natural) return ut_be_within is
+    l_result ut_matcher;
```

Comment:
use `l_result ut_be_within;`
that way there is no need for `return treat(... as )`